wallet/udb/txmined.go

```diff
@@ -1343,18 +1345,18 @@ func (s *Store) AddCredit(dbtx walletdb.ReadWriteTx, rec *TxRecord, block *Block
 
 // getP2PKHOpCode returns opNonstake for non-stake transactions, or
 // the stake op code tag for stake transactions.
-func getP2PKHOpCode(pkScript []byte) uint8 {
-	class := txscript.GetScriptClass(0, pkScript, true) // Yes treasury
+func getP2PKHOpCode(version uint16, pkScript []byte) uint8 {
+	class := stdscript.DetermineScriptType(version, pkScript)
 	switch class {
-	case txscript.StakeSubmissionTy:
+	case stdscript.STStakeSubmissionPubKeyHash:
```

Comment:
This function has **P2PKH** in its name, but looking at the callers I wonder if it should return opcodes for the P2SH variants of these stake script classes.   Please verify this.

Reply:
I've concluded the function name is just misleading.  Have updated the switch to include the script hash variants to keep existing behavior.